.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,5 +20,6 @@ This PR closes #{issue_number}.
 * [ ] I ran `task all`, and it passed.
 * [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
 * [ ] I checked comments rendering with `task godocs`.
+* [ ] I ensured that the title is good enough for the changelog.
```

Comment:
How can I understand that the title is good enough for the changelog?

Reply:
By using the best judgment :) I will fix it if it is bad. The point of this point is to prevent them from being obviously bad.